src/main/java/com/epam/test_generator/controllers/GlobalExceptionController.java

```diff
@@ -19,55 +21,66 @@
 @ControllerAdvice
 public class GlobalExceptionController {
 
+    private static final Logger logger = LoggerFactory.getLogger(GlobalExceptionController.class);
+
+
     @ExceptionHandler(Exception.class)
     public ResponseEntity<Void> handleRunTimeException(Exception ex) {
+
+        logger.error(ex.getMessage(), ex);
         return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
     }
 
     @ExceptionHandler(NotFoundException.class)
     public ResponseEntity<Void> handleNotFoundException(NotFoundException ex) {
+        logger.error(ex.getMessage(), ex);
```

Comment:
logger.warn is much more approptiate for 400+ error codes

Reply:
Done.